OrcScala/src/orc/error/OrcException.java

```diff
@@ -24,7 +24,7 @@
  *
  * @author dkitchin
  */
-public abstract class OrcException extends Exception {
+public abstract class OrcException extends RuntimeException {
```

Comment:
I'll look into making it checked. If I cant make it checked without having to wrap and unwrap the at truffle call boundaries I will. Otherwise I will make a TODO (and maybe an issue) and we can revisit it later.